src/heraldicon/frontend/element/tincture_modifiers.cljs

```diff
@@ -76,7 +76,7 @@
                                      (-> sanitized-tincture-data
                                          :highlight
                                          pos?)) (conj :highlight)))
-            tinctures-title (if (-> tinctures-set count pos?)
+            tinctures-title (if (not (s/blank? tinctures-set))
```

Comment:
Here's an opportunity to use `(if (s/blank? ...) ... ...)` now and swap the then/else for less mental gymnastics.